migrations/server/mysql/0004_drop_timestamp_key.up.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE `timestamp_keys`;
```

Comment:
I'd be more than happy to add a `IF EXISTS` to this drop table but I'm not sure if we should edit existing migrations?